lib/chronometer/dsl.rb

```diff
@@ -28,7 +28,7 @@ def method(method_name, **opts)
       opts[:method] = method_name
       opts[:event_type] ||= :X
       opts[:cls] = @cls || raise('must be in for_class block')
-      events << Event.new(opts)
+      events << Event.new(**opts)
```

Comment:
This was the real change we needed https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/